src/R/Components/Impl/ConnectionManager/IConnection.cs

```diff
@@ -8,8 +8,9 @@ namespace Microsoft.R.Components.ConnectionManager {
     public interface IConnection {
         Uri Id { get; }
         string Name { get; }
+        string Path { get; }
```

Comment:
The difference is that `Path` is what user has entered in UI. `string` -> `Uri` -> `string` conversion will change original input.

---
In reply to: [75948756](https://github.com/Microsoft/RTVS/pull/2192#discussion_r75948756) [](ancestors = 75948756)